File: backup/entities/Rate.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

#[ORM\Entity]
class Rate
{
    #[ORM\Id]
    #[ORM\GeneratedValue]
    #[ORM\Column(type: 'integer')]
    private ?int $id = null;

    #[ORM\ManyToOne(targetEntity: Agent::class, inversedBy: 'rates')]
    #[ORM\JoinColumn(name: 'agent_id', referencedColumnName: 'id')]
    private ?Agent $agent = null;

    #[ORM\Column(type: 'integer')]
    private ?int $rating = null;

    #[ORM\Column(type: 'text', nullable: true)]
    private ?string $comment = null;

    #[ORM\Column(type: 'datetime')]
    private ?\DateTimeInterface $rateDate = null;

    public function getId(): ?int
    {
        return $this->id;
    }

    public function getAgent(): ?Agent
    {
        return $this->agent;
    }

    public function setAgent(?Agent $agent): self
    {
        $this->agent = $agent;
        return $this;
    }

    public function getRating(): ?int
    {
        return $this->rating;
    }

    public function setRating(int $rating): self
    {
        $this->rating = $rating;
        return $this;
    }

    public function getComment(): ?string
    {
        return $this->comment;
    }

    public function setComment(?string $comment): self
    {
        $this->comment = $comment;
        return $this;
    }

    public function getRateDate(): ?\DateTimeInterface
    {
        return $this->rateDate;
    }

    public function setRateDate(\DateTimeInterface $rateDate): self
    {
        $this->rateDate = $rateDate;
        return $this;
    }
}

File: src/main/java/Models/Rate.java
package Models;

public class Rate {
    private int id;
    private int idAgent;
    private int rating;
    private String comment;
    private String sentiment;

    public Rate(int id, int idAgent, int rating, String comment, String sentiment) {
        this.id = id;
        this.idAgent = idAgent;
        this.rating = rating;
        this.comment = comment;
        this.sentiment = sentiment;
    }

    public int getId() { return id; }
    public int getIdAgent() { return idAgent; }
    public int getRating() { return rating; }
    public String getComment() { return comment; }
    public String getSentiment() { return sentiment; }

    public void setSentiment(String sentiment) {
        this.sentiment = sentiment;
    }

    @Override
    public String toString() {
        return "Rate{" +
                "id=" + id +
                ", idAgent=" + idAgent +
                ", rating=" + rating +
                ", comment='" + comment + '\'' +
                ", sentiment='" + sentiment + '\'' +
                '}';
    }


}
